### PR TITLE
US75672: add cisco_vrf:pt:new SILK1 attributes

### PR DESCRIPTION
**Pull request by HelenHZhang**
All beaker tests passed on eXR and n9kv.

### CODE REVIEW

File: lib/puppet/type/cisco_vrf.rb [hunk 1]
```diff
@@ -81,6 +81,32 @@ def self.title_patterns
     end
   end # property description
 
+  newproperty(:mhost_ipv4) do
```
Comment (mikewiebe) [inline lib/puppet/type/cisco_vrf.rb]: Please add these new properties to the doc example at the beginning of this file.

File: examples/cisco/demo_vrf.pp [hunk 2]
```diff
@@ -81,12 +96,21 @@
...
+
   cisco_vrf { 'test_vrf':
-    ensure              => present,
-    description         => 'test vrf for puppet',
-    route_distinguisher => $rd_auto,
-    shutdown            => $shutdown,
-    vni                 => $vni,
+    ensure                 => present,
+    description            => 'test vrf for puppet',
+    mhost_ipv4             => $mhost_intf,
```
Comment (chburket) [inline examples/cisco/demo_vrf.pp]: Looks like you missed these lines when the properties got renamed.

Comment (HelenHZhang) [inline examples/cisco/demo_vrf.pp]: fixed it, thx for catching it.

**Comment by chburket** [comment]
:+1:

**Comment by mikewiebe** [comment]
:+1:

**Pull request merged**